OWLib/Types/STUD/UnknownE76D9EC1.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.IO;
+using System.Runtime.InteropServices;
+
+namespace OWLib.Types.STUD {
+    [System.Diagnostics.DebuggerDisplay(OWLib.STUD.STUD_DEBUG_STR)]
+    public class UnknownE76D9EC1 : ISTUDInstance {
+        // There is currently only one of this file
+        // References a bunch of 003 (Game Logic)
+        public uint Id => 0xE76D9EC1;
+        public string Name => "UnknownE76D9EC1";
+
+        [StructLayout(LayoutKind.Sequential, Pack = 4)]
+        public struct UnknownE76D9EC1Data {
+            public STUDInstanceInfo instance;
+            public ulong arrayInfoOffset;  // 48
+            public ulong unk1;  // 0
+            public ulong unk2;  // 0
+            public ulong count;  // 7
```

Comment:
I'm assuming this is pointed to by arrayInfoOffset, don't include it in the header.